### ISSUE TITLE
Crashes on MetaKey pressing in Qt

### ISSUE THREAD

**Issue by mahaloz** [angr/angr-management#793]
### Description

Encountering a crash when hitting the Meta key in the decompilation view. May apply to other views as well. 

## Crash Log
```
Traceback (most recent call last):
  File "/home/mahaloz/github/angr-dev/angr-management/angrmanagement/ui/widgets/qccode_edit.py", line 222, in keyPressEvent
    xkey |= Qt.META
  File "/home/mahaloz/.virtualenvs/angr/lib/python3.10/site-packages/PySide6/support/deprecated.py", line 54, in func_or
    return QtCore.QKeyCombination(self, other)
TypeError: 'PySide6.QtCore.QKeyCombination.__init__' called with wrong argument types:
  PySide6.QtCore.QKeyCombination.__init__(Modifier, QKeyCombination)
Supported signatures:
  PySide6.QtCore.QKeyCombination.__init__(Union[PySide6.QtCore.QKeyCombination, PySide6.QtCore.Qt.KeyboardModifier, PySide6.QtCore.Qt.Key])
  PySide6.QtCore.QKeyCombination.__init__(PySide6.QtCore.Qt.Key = Instance(Qt.Key_unknown))
  PySide6.QtCore.QKeyCombination.__init__(PySide6.QtCore.Qt.KeyboardModifier, PySide6.QtCore.Qt.Key = Instance(Qt.Key_unknown))
  PySide6.QtCore.QKeyCombination.__init__(PySide6.QtCore.Qt.Modifier, PySide6.QtCore.Qt.Key)
```

## Steps
1. Open a binary 
2. Decompile it and be focused on the decompilation window
3. Hit the meta key on your keyboard while holding shift (the Window key)
4. Crash

### Environment

- Ubuntu 22.04
- i3 Window Manager (should be irrelevant)
- Latest angr suite

**Comment by rhelmot** [comment]
can't reproduce on ubuntu 22.04/gnome/latest angr

**Comment by mahaloz** [comment]
@rhelmot here are my current PySide6 versions:
```
PySide6            6.4.0.1
PySide6-Addons     6.4.0.1
PySide6-Essentials 6.4.0.1
PySide6-QtAds      3.8.3.1
``` 

I've explored this behavior further and found that **ANY** modifier combination breaks in the Decompilation window. I think this has something to do with a change in Qt.KeyboardModifier vs the old Qt.META, which is deprecated in PySide6 but can still work in some systems apparently.

**Comment by mahaloz** [comment]
@rhelmot I fixed for my system with the linked PR #795, would you be able to test it on your system, which I assume is a non-i3 variant of Ubuntu?
